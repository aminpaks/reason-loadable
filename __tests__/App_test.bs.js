

import * as Jest from "bs-jest/src/jest.js";
import * as ReasonReact from "reason-react/src/ReasonReact.js";
import * as ReactShallowRenderer from "bs-react-test-renderer/src/ReactShallowRenderer.bs.js";
import * as App$ReasonmlCodeSplit from "../src/App.bs.js";

describe("App", (function () {
        return Jest.test("render", (function () {
                      return Jest.Expect[/* toMatchSnapshot */16](Jest.Expect[/* expect */0](ReactShallowRenderer.renderWithRenderer(ReasonReact.element(/* None */0, /* None */0, App$ReasonmlCodeSplit.make(/* array */[])))));
                    }));
      }));

export {
  
}
/*  Not a pure module */
