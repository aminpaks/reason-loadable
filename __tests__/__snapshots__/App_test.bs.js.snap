// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render 1`] = `
<Layout
  reasonProps={
    Array [
      Array [
        "Layout",
        [Function],
        Array [
          0,
        ],
        [Function],
        [Function],
        [Function],
        [Function],
        [Function],
        [Function],
        [Function],
        [Function],
        0,
        [Function],
        [Function],
        0,
      ],
    ]
  }
/>
`;
